packages/wdio-runner/src/index.js

```diff
@@ -163,7 +157,7 @@ export default class Runner extends EventEmitter {
         /**
          * in watch mode we don't close the session and leave current page opened
          */
-        if (!argv.watch) {
+        if (!args.watch) {
```

Comment:
having this change in my local dev env of webdriverio makes my test project throw an error:

![joscha_Joschas-MacBook-Pro____work_wdio-ts-no-recompile_and_joscha_Joschas-MacBook-Pro____work_wdio-ts-no-recompile](https://user-images.githubusercontent.com/188038/73036964-58482000-3ea1-11ea-8db5-dbd666467c2e.png)

not sure how, yet.

Reply:
`git reset 1b11d192 --hard` makes the error go away.

Reply:
@joscha make sure you update all packages to `latest`

Reply:
I am running from within the repo via `npm run setup-full` and then I link the `@wdio/local-runner` package. With the code from a few days ago that works fine, with current `master` HEAD not anymore.

Reply:
you need to also link the `@wdio/cli` package because it has changes that are required for the new runner version to work.

Reply:
I see; I assumed if there is no major version bump in either, these packages should be compatible to each other. Thanks!

Reply:
No, semver is applied to user facing changes not internal ones.